k8s/cc-global-api-db.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cc-db-deploy
+  labels:
+    app: db
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: db
+  template:
+    metadata:
+      labels:
+        app: db
+    spec:
+      volumes:
+        - name: postgres
+          persistentVolumeClaim:
+            claimName: cc-db-pvc
```

Comment:
This might need to be different, so the global-api DB doesn't use the volume for the CityCatalyst DB